### PR TITLE
APICLIENT-14: Remove numeric ids

### PR DESCRIPTION
**Pull request by maxammann**
This is planned to be merged after the addition of the hash property.

### CODE REVIEW

File: src/endpoints/createCategoriesEndpoint.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default (baseUrl: string): Endpoint<ParamsType, CategoriesMapModel> =>
       const categories = json
         .map(category => {
           return new CategoryModel({
-            id: category.id,
+            root: category.id === 0,
```
Comment (NKlug) [inline src/endpoints/createCategoriesEndpoint.js]: This is always `false` as far as I can see, so we can hard code it.

File: src/endpoints/createCategoriesEndpoint.js [hunk 2]
```diff
@@ -37,20 +37,22 @@ export default (baseUrl: string): Endpoint<ParamsType, CategoriesMapModel> =>
...
         path: basePath,
         title: params.city,
         parentPath: '',
         content: '',
         thumbnail: '',
         order: -1,
         availableLanguages: new Map(),
-        lastUpdate: moment(0)
+        lastUpdate: moment(0),
+        hash: ''
```
Comment (NKlug) [inline src/endpoints/createCategoriesEndpoint.js:55]: Do we have any problems if the hash is empty?

**Review by steffenkleinle — APPROVED** [review]

File: src/endpoints/createCategoriesEndpoint.js (hunk 2 above)
Comment (maxammann) [reply src/endpoints/createCategoriesEndpoint.js:55]: I don't think so. The comparision of hashes should be a simple strcmp

File: src/endpoints/createCategoriesEndpoint.js (hunk 1 above)
Comment (maxammann) [reply src/endpoints/createCategoriesEndpoint.js]: oh you are right! lolz

**Pull request merged**